finetune/multiple_choice.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+
+from finetune.base import BaseModel
+from finetune.encoding import ArrayEncodedOutput
+from finetune.target_encoders import IDEncoder
+import tensorflow as tf
+
+from finetune.network_modules import multi_choice_question
+from finetune.utils import list_transpose
+
+
+class MultipleChoice(BaseModel):
+    """
+    Multi choice question finetune model.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.num_answers = None
+
+    def _text_to_ids(self, question, answers, Y=None, max_length=None):
+        """
+        Format multi question examples as a list of IDs
+        """
+        arrays = [super(MultipleChoice, self)._text_to_ids(question, ans) for ans in answers]
+        kwargs = arrays[0]._asdict()
+        kwargs['tokens'] = [arr.tokens for arr in arrays]
+        kwargs['token_ids'] = np.stack([arr.token_ids for arr in arrays], 1)
+        kwargs['mask'] = np.stack([arr.mask for arr in arrays], 1)
+        return ArrayEncodedOutput(**kwargs)
+
+    def finetune(self, question, correct_answer, answers, batch_size=None, fit_lm_only=False):
```

Comment:
Swap the order of answers and correct_answers (so it's like inputs, target) and this is good to ship.